deploy/example_catalog/cr-app-mapr610-secured.json

```diff
@@ -0,0 +1,340 @@
+{
+  "apiVersion": "kubedirector.hpe.com/v1beta1",
+  "kind": "KubeDirectorApp",
+  "metadata": {
+    "name": "mapr610"
+  },
+  "spec": {
+    "systemdRequired": true,
+    "defaultPersistDirs": [
+      "/opt/mapr"
+    ],
+    "config": {
+      "configMeta": {
+        "storage": "60",
+        "secure": "true"
+      },
+      "roleServices": [
+        {
+          "roleID": "control-system",
+          "serviceIDs": [
+            "warden",
+            "mapr-cs",
+            "ssh"
+          ]
+        },
+        {
+          "roleID": "cldb",
+          "serviceIDs": [
+            "warden",
+            "cldb",
+            "ssh"
+          ]
+        },
+        {
+          "roleID": "zookeeper",
+          "serviceIDs": [
+            "warden",
+            "zookeeper",
+            "ssh"
+          ]
+        },
+        {
+          "roleID": "hive-server2",
+          "serviceIDs": [
+            "warden",
+            "hive-server2",
+            "webhcat",
+            "ssh"
+          ]
+        },
+        {
+          "roleID": "hive-meta",
+          "serviceIDs": [
+            "warden",
+            "hive-meta",
+            "mysql",
+            "ssh"
+          ]
+        },
+        {
+          "roleID": "hue",
+          "serviceIDs": [
+            "warden",
+            "hue",
+            "httpfs",
+            "ssh"
+          ]
+        },
+        {
+          "roleID": "edge",
+          "serviceIDs": [
+            "mapr-client",
+            "ssh"
+          ]
+        },
+        {
+          "roleID": "nodemanager",
+          "serviceIDs": [
+            "warden",
+            "yarn-nm",
+            "ssh"
+          ]
+        },
+        {
+          "roleID": "resource-manager",
+          "serviceIDs": [
+            "warden",
+            "yarn-rm",
+            "ssh"
+          ]
+        },
+        {
+          "roleID": "history-server",
+          "serviceIDs": [
+            "warden",
+            "yarn-hs",
+            "ssh"
+          ]
+        },
+        {
+          "roleID": "fileserver",
+          "serviceIDs": [
+            "warden",
+            "fileserver",
+            "ssh"
+          ]
+        }
+      ],
+      "selectedRoles": [
+        "control-system",
+        "resource-manager",
+        "history-server",
+        "zookeeper",
+        "cldb",
+        "nodemanager",
+        "fileserver"
```

Comment:
Until we introduce choice-driven config variations this probably won't matter much and will seem like redundant busywork, but... the selectedRoles here should list all of the defined roles in the app.

Reply:
i have updated the json file to include all roles under "selectedRoles"